collector/src/main/java/com/bitmovin/analytics/data/Backend.kt

```diff
@@ -0,0 +1,8 @@
+package com.bitmovin.analytics.data
+
+import okhttp3.Callback
+
+interface Backend {
+    fun send(eventData: EventData, callback: Callback?)
```

Comment:
is there another way this could be done? the `RetryBackend` has this parameter as well but will just ignore it -  I would expect the callback to be called though.
So maybe the `inner` backend for the `RetryBackend` should actually be a `HttpBackend`, and have those 2 methods, or we should implement it properly in the `RetryBackend` as well.